students/chris_lombardi/Lesson04/mailroom_part2.py

```diff
@@ -69,10 +69,10 @@ def send_letters():
 
 def thankyou_note(entry):
     note = (f'Dear {entry},\n\n\tThank you for your generous donation of '
-          f'${sum(donor_log.get(entry)):,.2f}!\n\tWe appreciate the '
-          f'{len(donor_log.get(entry)):d} total donation(s) that you have made.'
-          '\n\tYour donation will be put to good use.'
-          '\n\n\tSincerely,\n\t-The Mailroom Team')
+            f'${sum(donor_log.get(entry)):,.2f}!\n\tWe appreciate the '
+            f'{len(donor_log.get(entry)):d} total donation(s) that you have made.'
+            '\n\tYour donation will be put to good use.'
+            '\n\n\tSincerely,\n\t-The Mailroom Team')
```

Comment:
since your code has already been merged for mailroom I can leave all comments in one block here:
* line 68: `f.close()` no need to close since `with` handles it for you
* line 41: use `sum()` function instead of manually adding donations
* line 28: create_report function is missing sorting feature
* line 92: no need to use `set` or extract into variable -> `if option not in dict_menu_opts` is the same